eth/config.go

```diff
@@ -60,6 +60,7 @@ var DefaultConfig = Config{
 		Blocks:     20,
 		Percentile: 60,
 	},
+	Checkpoint: params.MainnetTrustedCheckpoint,
```

Comment:
I'm not sure about this one. I think I'd keep the default as `nil`, and set it inside `eth` by default if we have one available for the network. Otherwise everyone using go-ethereum as a library will need one more thing to explicitly add.